docs/component-implementation-2.md

```diff
@@ -256,6 +256,18 @@ fprime-util build
 ```
 > Resolve any errors and finish any TODOs before continuing.
 
+## Try it yourself
```

Comment:
Same here replacing line 49.  I do like that you've called out these other things I missed....but the existing call-outs are hard to see.

Reply:
Done. I also missed it.